configure-c8y.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
```

Comment:
It is good to stop on unexpected errors.

```suggestion
set -e
```